pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/GenerateDataCommand.java

```diff
@@ -120,9 +123,9 @@ public boolean execute()
     gen.init(spec);
 
     if (FORMAT_AVRO.equalsIgnoreCase(_format)) {
-      DataGenerationHelpers.generateAvro(gen, _numRecords, _numFiles, _outDir, _overwrite);
+      DataGenerationHelpers.generateAvro(gen, _numRecords, _numFiles, _outDir, _overwrite, _fileIndex);
     } else if (FORMAT_CSV.equalsIgnoreCase(_format)) {
-      DataGenerationHelpers.generateCsv(gen, _numRecords, _numFiles, _outDir, _overwrite);
+      DataGenerationHelpers.generateCsv(gen, _numRecords, _numFiles, _outDir, _overwrite, _fileIndex);
```

Comment:
Why is this not applicable to the JSON generator?

Reply:
Missed a spot! Added